internal/nonodo/nonodo.go

```diff
@@ -393,10 +395,27 @@ func NewSupervisor(opts NonodoOpts) supervisor.SupervisorWorker {
 	}
 
 	if opts.AvailEnabled {
+		paioLocation := ""
+
```

Comment:
here we could have a method to resolve this paioLocation
```golang
paioLocation := paiodecoder.RequirePaioLocation()
```

Reply:
how to work this method?